### PR TITLE
Allow the DAO to reduce the trading liquidity of a pool in a safe and controlled manner

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

File: contracts/pools/PoolCollection.sol [hunk 1]
```diff
@@ -207,6 +208,17 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
      */
     event DepositLimitUpdated(Token indexed pool, uint256 prevDepositLimit, uint256 newDepositLimit);
 
+    /**
+     * @dev triggered when a pool's trading liquidity is reduced
+     */
+    event TradingLiquidityReduced(
```
Comment (yudilevi) [inline contracts/pools/PoolCollection.sol:214]: There's already the TradingLiquidityUpdated event that must be fired when the trading liquidity is updated

File: contracts/pools/PoolCollection.sol [hunk 2]
```diff
@@ -564,6 +576,53 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
...
+        if (bntAmount >= bntTradingLiquidity) {
+            return;
+        }
+
+        if (bntAmount == 0) {
+            _resetTradingLiquidity(bytes32(0), pool, data, TRADING_STATUS_UPDATE_ADMIN);
+            return;
+        }
+
+        if (bntAmount < _networkSettings.poolFundingLimit(pool)) {
```
Comment (yudilevi) [inline contracts/pools/PoolCollection.sol:599]: would be helpful to have some comments throughout the code to provide the rationale for these conditions, especially this one

File: contracts/pools/PoolCollection.sol (hunk 1 above)
Comment (barakman) [reply contracts/pools/PoolCollection.sol:214]: So you want to replace the emitting of this event, with emitting the `TradingLiquidityUpdated` event?

Comment (barakman) [reply contracts/pools/PoolCollection.sol]: Fixed

File: contracts/pools/PoolCollection.sol (hunk 2 above)
Comment (barakman) [reply contracts/pools/PoolCollection.sol:582]: Done

**Pull request closed without merging**